acceptance/bundle/templates/telemetry/custom-template/script

```diff
@@ -0,0 +1,14 @@
+trace $CLI bundle init . --config-file input.json --output-dir output
+
+# We only need to keep the output databricks.yml to assert the bundle_uuid is consistent
+# in both the generated YAML file and telemetry payload.
```

Comment:
If we read the UUID from `databricks.yml`, to then replace it in `out.requests.txt`, then do we really need to keep it around?